reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -375,6 +375,19 @@
 		return onAssembly(new FluxConcatIterable<>(sources));
 	}
 
+	/**
+	 * Concatenate all values provided as a vararg.
+	 *
+	 * @param values The values to concatenate
+	 *
+	 * @return a new {@link Flux} concatenating all source sequences
+	 * @since 3.1.2
+	 */
+	@SafeVarargs
+	public final Flux<T> concatWith(T... values) {
```

Comment:
that one is interesting. I wonder if naming it `endWith(T...)` would make sense, as a dual to `startWith(T...)`